test/model/store_test.dart

```diff
@@ -153,12 +154,12 @@ void main() {
 
     test('reject changing id, realmUrl, or userId', () async {
       final globalStore = eg.globalStore(accounts: [eg.selfAccount]);
-      check(globalStore.updateAccount(eg.selfAccount.id, const AccountsCompanion(
-        id: Value(1234)))).throws();
-      check(globalStore.updateAccount(eg.selfAccount.id, AccountsCompanion(
-        realmUrl: Value(Uri.parse('https://other.example'))))).throws();
-      check(globalStore.updateAccount(eg.selfAccount.id, const AccountsCompanion(
-        userId: Value(1234)))).throws();
+      finish(check(globalStore.updateAccount(eg.selfAccount.id, const AccountsCompanion(
+        id: Value(1234)))).throws());
```

Comment:
For `throws`, I think the normal thing is to await the future:
```suggestion
      await check(globalStore.updateAccount(eg.selfAccount.id, const AccountsCompanion(
        id: Value(1234)))).throws();
```

From the [docs](https://github.com/dart-lang/test/tree/8e8a83607d90a7a6813fa378b2d1962a2fc0d44b/pkgs/checks#checking-asynchronous-expectations) linked in the commit message:
> Expectation extension methods checking asynchronous behavior return a `Future`. The future should typically be awaited within the test body, […]